std_msgs/Header header 
cv_lidar_fusion_pkg/relative_obstacle[] obstacle_array